### PR TITLE
Add RoboFEI logos and info

### PR DESCRIPTION
**Pull request by leonardoSCosta**
Adicionando logo vertical, horizontal, ícone e informações sobre o time.

### CODE REVIEW

File: robofei/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# RoboFEI
+### Team information
```
Comment (FelipeGdM) [inline robofei/README.md:2]:
```suggestion
## Team information
```

File: robofei/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- Name: RoboFEI
+- USUAL NAME: ROBOFEI
```
Comment (FelipeGdM) [inline robofei/README.md:4]:
```suggestion
- Usual name: ROBOFEI
```

**Review by FelipeGdM — CHANGES_REQUESTED** [review]
Some adjusts are needed in the README.md

File: robofei/README.md (hunk 1 above)
Comment (leonardoSCosta) [reply robofei/README.md]: Ok

File: robofei/README.md (hunk 2 above)
Comment (leonardoSCosta) [reply robofei/README.md]: Ok

**Review by FelipeGdM — APPROVED** [review]

**Pull request merged**